package.json

```
@@ -13,7 +13,7 @@
     "clean": "lerna clean --yes && rimraf node_modules",
     "commit": "cz",
     "nx": "nx",
-    "lint-staged": "lint-staged",
+    "lint-staged": "NODE_OPTIONS=--max_old_space_size=8192 lint-staged",
```

Without this committing would fail with the following error:
```
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
```
I aligned this value with the ones present in `apps/web`

---

This could be an indication of a memory leak, we shouldn't need this much memory to run a linting job. In the past when this has occurred, restarting my machine fixes the issue. I'm hesitant to make this change because it may mask further memory leaks. I'd suggest we revert for now.

```suggestion
    "lint-staged": "lint-staged",
```

---

reverted back